docs/additional/interesting_aspects_en.md

```diff
@@ -0,0 +1,249 @@
+This includes aspects like:
+- Checking permission for scripts (startup.sh), volumes.
+- Checking the UID & GID for files (startup.sh, resources, volumes, write-dependent paths)
+- Accessibility of the software: TCP health checks
+
+For pipeline integration, we recommend using the `ecoSystem.verify()` method from the [dogu-build-lib][dogu-build-lib].  
+An example call looks like the following in a Jenkinsfile:
+
+```groovy
+stage('Verify') {
+    ecoSystem.verify("/dogu")
+}
+```
+
+This internally calls the `cesapp verify` command, which runs the goss tests.
+
+[goss]: https://github.com/goss-org/goss
+[dogu-build-lib]: https://github.com/cloudogu/dogu-build-lib
+
+## Documentation
+
+Every tool should have a reasonable documentation about used features, which supports usage or support.
+To allow easy integration into our existing documentation infrastructure, the documentation should be placed in a `/docs` folder located in the repository root.
+
+By default, the docs include the following:
+- Features of the Dogu
+- Installation of the Dogu
+- Configuration of the Dogu
+- Development on the Dogu
+- System requirements of the Dogu (nice to have)
+- Readme
+- Changelog
+
+### License
+
+The license for using the Dogu should be clearly stated.
+This can be specified in the Readme, for example.
+
+### Readme
+
+A Readme is useful both for your own developments and in a partner context with external developers.  
+In the case of public repositories, however, a higher standard should be applied to a Readme.
+
+A Readme file is located in the repository root and concisely describes notes on the general purpose of the Dogu.
+This can be in the form of a quickstart guide, for example, describing how to set up a local development environment.
+It should also include a reference to the documentation folder and can also refer to specific pages of the documentation.
+Finally, it is recommended to specify required resources for use and a person responsible for the Dogu.
+
+### Changelog
+
+Changelogs are written for people who want to understand what's new in releases.
+
+The format of the changelog should follow **[keep a changelog](https://keepachangelog.com)**.
+
+New changes are always added to the top section `## [Unreleased]`.  
+When a new release is made, these changes move to a new section containing the corresponding version number.
+
+For all changes, references to the operations of the issue tracker used should be added.  
+In addition, if more complex procedures are involved, a reference to documentation is useful.
+
+### System requirements
+
+The system requirements of a Dogu (CPU, RAM, storage) can either be recorded in the Dogu's documentation
+or should be otherwise communicated to Cloudogu directly (e.g. in case of a private repository).
+
+## Release of a Dogus
+
+### Cloudogu account and permissions
+
+First, you need to create an account on https://account.cloudogu.com.  
```

Comment:
This paragraph should mention that this step is only necessary once per devleoper account.